contracts/okp4-objectarium/src/state.rs

```diff
@@ -208,7 +208,7 @@ impl From<BucketLimits> for msg::BucketLimits {
 /// Pagination is the type carrying configuration for paginated queries.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct Pagination {
-    /// The maximum elements a page can contains.
+    /// The maximum elements a page can contain.
```

Comment:
It seems some changes on the `objectarium` are not present in the generated docs, maybe you can run the generation script to ensure it comply:

```bash
cargo make schema
cargo make docs-generate
```

Reply:
@amimart I'm getting some errors with the generation, just edited the file to match, I hope that's ok. (I'm quite a noob 🙂)